tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -33,19 +34,29 @@ export const nhsAppRadioSelectedSessionData: Session = {
   nhsAppTemplateMessage: '',
 };
 
+const testUserEmail = 'choose-page@nhs.net';
```

Comment:
As per previous comment, lets include `nhs-notify-automated-test` as a prefix in case we end up sending something.

Reply:
see other response